package.json

```diff
@@ -25,6 +25,7 @@
     "body-parser": "^1.18.2",
     "cheerio": "^1.0.0-rc.2",
     "expres": "0.0.5",
```

Comment:
Need to make sure you're installing the correct packages. Not sure what `expres` does, but my guess is they were trying to get a bunch of downloads by naming their package very close to `express`